core/domain/platform_parameter_list.py

```diff
@@ -54,6 +54,7 @@ class ParamName(enum.Enum):
     UNPUBLISH_EXPLORATION_EMAIL_HTML_BODY = (
         'unpublish_exploration_email_html_body')
     RECORD_PLAYTHROUGH_PROBABILITY = 'record_playthrough_probability'
+    CAN_SEND_EMAILS = 'can_send_emails'
```

Comment:
I think the old feconf name for this constant is a bit dated (we had far fewer configuration values when it was originally introduced). It's not super clear from the name who the 'can' is in this expression.

Perhaps something like 'SERVER_CAN_SEND_EMAILS' or 'ENABLE_EMAIL_SERVICE' might be clearer?

Reply:
Just a note -- prefer the first constantname suggestion, since it's not clear what the "email service" is (since both the frontend and backend have a concept of services).